Index: .idea/dictionaries/erny.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/dictionaries/erny.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ .idea/dictionaries/erny.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="erny">
+    <words>
+      <w>usuario</w>
+    </words>
+  </dictionary>
+</component>
\ No newline at end of file
Index: app/src/main/java/company/viral/organizadorjec/Fragment/AjustesF.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/Fragment/AjustesF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/java/company/viral/organizadorjec/Fragment/AjustesF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,20 @@
+package company.viral.organizadorjec.Fragment;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import company.viral.organizadorjec.R;
+
+
+public class AjustesF extends Fragment {
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_ajustes, container, false);
+    }
+
+}
Index: app/src/main/java/company/viral/organizadorjec/Fragment/CalendarioF.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/Fragment/CalendarioF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/java/company/viral/organizadorjec/Fragment/CalendarioF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,20 @@
+package company.viral.organizadorjec.Fragment;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import company.viral.organizadorjec.R;
+
+
+public class CalendarioF extends Fragment {
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_calendario, container, false);
+    }
+
+}
Index: app/src/main/java/company/viral/organizadorjec/Fragment/InicioF.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/Fragment/InicioF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/java/company/viral/organizadorjec/Fragment/InicioF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,78 @@
+package company.viral.organizadorjec.Fragment;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+import company.viral.organizadorjec.R;
+
+import static company.viral.organizadorjec.R.id.menu15dias;
+
+
+public class InicioF extends Fragment {
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        View view = inflater.inflate(R.layout.fragment_inicio, container, false);
+
+        String[] menuDias = {   "Tare",
+                                "Trabajo",
+                                "Salida","Tare",
+                "Trabajo",
+                "Salida"};
+
+        String[] menu15Das = {  "Tare",
+                                "Trabajo",
+                                "Salida"};
+
+        String[] menumes = {    "Tare",
+                                "Trabajo",
+                                "Salida",
+                                "paseo de perro","comer"};
+
+
+
+        //adaptadores
+        //adaptador dias
+        ListView listdias = (ListView) view.findViewById(R.id.menudias);
+
+        ArrayAdapter<String> listavistadias = new ArrayAdapter<String>(
+                getActivity(),
+                android.R.layout.simple_list_item_1,menuDias);
+
+
+        listdias.setAdapter(listavistadias);
+
+        //adaptador 15dias
+
+        ListView lista15dias = (ListView) view.findViewById(R.id.menu15dias);
+
+        ArrayAdapter<String> listavista15dias = new ArrayAdapter<String>(
+                getActivity(),
+                android.R.layout.simple_list_item_1,menu15Das);
+
+        lista15dias.setAdapter(listavista15dias);
+
+        //adaptador mes
+
+        ListView listames = (ListView) view.findViewById(R.id.menumes);
+
+        ArrayAdapter<String> listavistames =new ArrayAdapter<String>(
+                getActivity(),
+                android.R.layout.simple_list_item_1,menumes);
+
+        listames.setAdapter(listavistames);
+
+        return view ;
+    }
+
+}
Index: app/src/main/java/company/viral/organizadorjec/Fragment/PerfilF.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/Fragment/PerfilF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/java/company/viral/organizadorjec/Fragment/PerfilF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,20 @@
+package company.viral.organizadorjec.Fragment;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import company.viral.organizadorjec.R;
+
+
+public class PerfilF extends Fragment {
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_perfil, container, false);
+    }
+
+}
Index: app/src/main/java/company/viral/organizadorjec/Fragment/ProfesoresF.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/Fragment/ProfesoresF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/java/company/viral/organizadorjec/Fragment/ProfesoresF.java	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,20 @@
+package company.viral.organizadorjec.Fragment;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import company.viral.organizadorjec.R;
+
+
+public class ProfesoresF extends Fragment {
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_profesores, container, false);
+    }
+
+}
Index: app/src/main/res/layout/fragment_ajustes.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_ajustes.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/res/layout/fragment_ajustes.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,20 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="company.viral.organizadorjec.Fragment.InicioF"
+    android:background="#e4b48b">
+
+
+    <!-- TODO: Update blank fragment layout -->
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ListView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+    </LinearLayout>
+</FrameLayout>
Index: app/src/main/res/layout/fragment_Calendario.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_Calendario.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/res/layout/fragment_Calendario.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,20 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="company.viral.organizadorjec.Fragment.InicioF"
+    android:background="#8fbab5">
+
+
+    <!-- TODO: Update blank fragment layout -->
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ListView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+    </LinearLayout>
+</FrameLayout>
Index: app/src/main/res/layout/fragment_inicio.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_inicio.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/res/layout/fragment_inicio.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,58 @@
+
+
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="company.viral.organizadorjec.Fragment.InicioF">
+
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+
+        <TextView
+                        android:text="Proximos Dias"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"/>
+
+
+        <ListView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:id="@+id/menudias"
+            android:layout_weight="1"/>
+
+        <TextView
+                        android:text="Proximos 15 Dias"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"/>
+
+
+
+
+        <ListView
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:id="@+id/menu15dias"
+            android:layout_weight="1"/>
+
+
+                <TextView
+                        android:text="Proximo Mes"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"/>
+
+                    <ListView
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:id="@+id/menumes"
+                        android:layout_weight="1"/>
+
+
+    </LinearLayout>
+
+
+</FrameLayout>
Index: app/src/main/res/layout/fragment_perfil.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_perfil.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/res/layout/fragment_perfil.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,20 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="company.viral.organizadorjec.Fragment.InicioF"
+    android:background="#99d0e1">
+
+
+    <!-- TODO: Update blank fragment layout -->
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ListView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+    </LinearLayout>
+</FrameLayout>
Index: app/src/main/res/layout/fragment_profesores.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/fragment_profesores.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
+++ app/src/main/res/layout/fragment_profesores.xml	(revision 9c5fc10058fbfb73f2caf107c0291a74f9dcb627)
@@ -0,0 +1,28 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="company.viral.organizadorjec.Fragment.ProfesoresF">
+
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+            android:text="Listado de Profesores"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:id="@+id/textView2"
+            android:gravity="center"
+            android:textSize="24sp"
+            tools:textColor="@android:color/holo_blue_dark" />
+
+        <ListView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:id="@+id/listprofe"/>
+    </LinearLayout>
+
+</FrameLayout>
Index: app/src/main/java/company/viral/organizadorjec/AyudaBD.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/AyudaBD.java	(date 1479146651000)
+++ app/src/main/java/company/viral/organizadorjec/AyudaBD.java	(date 1480449937000)
@@ -11,6 +11,7 @@
  */
 
 public class AyudaBD extends SQLiteOpenHelper{
+
     /*constructor de la base de datos */
     /*cargo directamente desde el super*/
     /* (context cuando llamamos a la clase, el nombre de la base de datos, factory queda igual, version=1*/
Index: app/src/main/java/company/viral/organizadorjec/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/MainActivity.java	(date 1479146651000)
+++ app/src/main/java/company/viral/organizadorjec/MainActivity.java	(date 1480449937000)
@@ -11,52 +11,80 @@
 import android.widget.TextView;
 import android.widget.Toast;
 
+//aqui empieza...
 public class MainActivity extends AppCompatActivity {
 
+    //creamos variables EditText para capturar los datos
     private EditText aetid,aetpass;
-    private Button abtnacep,abtnreg;
+    private Cursor fila;
 
+
+    //en este metodo SIEMPRE se dibuja la app correspondiente
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+    //antes de dibujar definimos las variables y a quienes pertecen en el layout
+
         aetid = (EditText) findViewById(R.id.etid);
         aetpass = (EditText) findViewById(R.id.etpass);
-        abtnacep = (Button) findViewById(R.id.btnacep);
-        abtnreg = (Button) findViewById(R.id.btnreg);
+
     }
 
+
+
+
+
+    //creamos los metodos con los que reaccionan los btn (onClick)
     /*metodo para entrar y buscar (en construccion.... explorando metodos)*/
     public void onClickAceptar (View view) {
 
         String auxn = aetid.getText().toString();
         String auxp = aetpass.getText().toString();
 
-        AyudaBD db = new AyudaBD(getApplicationContext(),null,null,1);
-        String buscar = new String();
+        SQLite admin = new SQLite(this,"administracion", null, 1);
+        SQLiteDatabase bd = admin.getWritableDatabase();
 
+        fila=bd.rawQuery("select usuario, clave from usuarios where usuario='"+auxn+"'and clave='"+auxp+"'",null);
 
-    /*validamos la cargada de datos*/
-        if (auxn.isEmpty()) {
 
-            Toast.makeText(getApplicationContext(), "Ingrese un Usuario", Toast.LENGTH_LONG).show();
 
-        } else if (auxp.isEmpty()){
+        if(fila.moveToFirst()==true){
 
-            Toast.makeText(getApplicationContext(), "Ingrese clave", Toast.LENGTH_LONG).show();
-        } else {
+            //capturamos los valores del cursos y lo almacenamos en variable
+            String usua=fila.getString(0);
+            String pass=fila.getString(1);
 
+            //preguntamos si los datos ingresados son iguales
+            if (auxn.equals(usua)&&auxp.equals(pass)){
 
-            Intent i = new Intent(this,Menusito.class);
-            startActivity(i);
+                //si son iguales entonces vamos a otra ventana
+                //Menu es una nueva actividad empty
+                Intent ven=new Intent(this,MenuCentral.class);
+
+                startActivity(ven);
+
+                //limpiamos las las cajas de texto
+                aetid.setText("");
+                aetpass.setText("");
+
-        }
+            }
+
+        }else {
+
+            Toast.makeText(getApplicationContext(), "Usuario o contraseña erroneo", Toast.LENGTH_LONG).show();
-    }
+        }
+
+        bd.close();
+    }
+
+
+    //metodo para entrar a la actividad de registro
+
     public void onClickRegistro(View view){
         Intent i = new Intent(this,Registro.class);
         startActivity(i);
-
     }
-
 
 }
Index: app/src/main/java/company/viral/organizadorjec/Menusito.java
===================================================================
--- app/src/main/java/company/viral/organizadorjec/Menusito.java	(date 1479146651000)
+++ app/src/main/java/company/viral/organizadorjec/MenuCentral.java	(date 1480449937000)
@@ -3,6 +3,7 @@
 import android.os.Bundle;
 import android.support.design.widget.FloatingActionButton;
 import android.support.design.widget.Snackbar;
+import android.support.v4.app.FragmentManager;
 import android.view.View;
 import android.support.design.widget.NavigationView;
 import android.support.v4.view.GravityCompat;
@@ -13,16 +14,27 @@
 import android.view.Menu;
 import android.view.MenuItem;
 
-public class Menusito extends AppCompatActivity
+import company.viral.organizadorjec.Fragment.AjustesF;
+import company.viral.organizadorjec.Fragment.CalendarioF;
+import company.viral.organizadorjec.Fragment.InicioF;
+import company.viral.organizadorjec.Fragment.PerfilF;
+import company.viral.organizadorjec.Fragment.ProfesoresF;
+
+public class MenuCentral extends AppCompatActivity
         implements NavigationView.OnNavigationItemSelectedListener {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_menusito);
+        setContentView(R.layout.activity_menu_central);
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
+        //colocamos el fragment con que inicia el menu
+
+        FragmentManager fragmentManager = getSupportFragmentManager();
+        fragmentManager.beginTransaction().replace(R.id.contenedor,new InicioF()).commit();
+
         FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
         fab.setOnClickListener(new View.OnClickListener() {
             @Override
@@ -55,7 +67,7 @@
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         // Inflate the menu; this adds items to the action bar if it is present.
-        getMenuInflater().inflate(R.menu.menusito, menu);
+        getMenuInflater().inflate(R.menu.menu_central, menu);
         return true;
     }
 
@@ -80,16 +92,18 @@
         // Handle navigation view item clicks here.
         int id = item.getItemId();
 
+        FragmentManager fragmentManager = getSupportFragmentManager();
+
         if (id == R.id.nav_camera) {
-            // Handle the camera action
+            fragmentManager.beginTransaction().replace(R.id.contenedor,new InicioF()).commit();
         } else if (id == R.id.nav_gallery) {
-
+            fragmentManager.beginTransaction().replace(R.id.contenedor,new ProfesoresF()).commit();
         } else if (id == R.id.nav_slideshow) {
-
+            fragmentManager.beginTransaction().replace(R.id.contenedor,new CalendarioF()).commit();
         } else if (id == R.id.nav_manage) {
-
+            fragmentManager.beginTransaction().replace(R.id.contenedor,new AjustesF()).commit();
         } else if (id == R.id.nav_share) {
-
+            fragmentManager.beginTransaction().replace(R.id.contenedor,new PerfilF()).commit();
         } else if (id == R.id.nav_send) {
 
         }
Index: app/src/main/java/company/viral/organizadorjec/Registro.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/Registro.java	(date 1479146651000)
+++ app/src/main/java/company/viral/organizadorjec/Registro.java	(date 1480449937000)
@@ -29,37 +29,108 @@
     }
 
     public void onClickAceptar (View view) {
-        /*metodo utilizado para guardar en la BD */
-        String auxn = aetidr.getText().toString();
-        String auxp = aetpassr.getText().toString();
-        String auxc = aetpasscr.getText().toString();
-        /*validaciones correspondientes para el registro de usuario*/
+        //metodo utilizado para guardar en la BD
+        //1)las variables usadas para llenar la tabla usuario
+
+        String auxn = aetidr.getText().toString();  //tomamos el nombre
+        String auxp = aetpassr.getText().toString();//tomamos la clave
+        String auxc = aetpasscr.getText().toString();//con este validamos la clave
+
+        //verificamos QUE NADA QUEDE SIN LLENAR!!!
+
+        //si la variable que acepta el nombre esta vacia nos pedira ingresar usuario
         if (auxn.isEmpty()){
 
             Toast.makeText(getApplicationContext(),"Ingrese un Usuario",Toast.LENGTH_LONG).show();
 
+         //sino... si la clave es vacia entonces te pide que coloques una clave
+        }else if (auxp.equals("")){
+
+            Toast.makeText(getApplicationContext(),"Ingrese una contraseña",Toast.LENGTH_LONG).show();
+
+
+            //si la clave de confirmacion es igual a la clave entonces guardalo en el sistema
+        }else if (auxp.equals(auxc)){
+
+            //abrimos la base de datos
+            SQLite admin = new SQLite(this,"administracion", null, 1);
+
+            //creamos variable re-escribible
+            SQLiteDatabase bd = admin.getWritableDatabase();
+
+            //creamos un contenedor llamado registro
+            ContentValues registro = new ContentValues();
+
+            //cargamos los datos en el contenedor..
+            //diciendo... "registro se cargue con variable "usuario" y el valor de auxn
+            registro.put("usuario", auxn);
+            //diciendo... "registro se cargue con variable "clave" y el valor de auxn
+            registro.put("clave", auxp);
+            //estas variables deben ser EXACTAS a la de la tabla!
+
+
+            // los inserto en la base de datos diciendo:
+            //variable db (creada arriba) insertara los valores EN la tabla "usuarios", null, con el contenido de "registro"
+
+            bd.insert("usuarios", null, registro);
+
+            //cerramos bd
+            bd.close();
+
+            // ponemos los campos a vacío para insertar el siguiente usuario
+            aetidr.setText("");
+            aetpasscr.setText("");
+            aetpassr.setText("");
+
+            Toast.makeText(this, "Datos del usuario cargados", Toast.LENGTH_SHORT).show();
+
+
+
+            //de ser incorrecto solo queda que las claves no coinciden..
+        }else {
+        Toast.makeText(getApplicationContext(),"La confirmacion no coincide",Toast.LENGTH_LONG).show();
+    }
+
+
+
+
+
+
+}
+
+        /*PRUEBA CON SQLITE
+        //validaciones correspondientes para el registro de usuario
+
+        //si la variable que acepta el nombre esta vacia nos pedira ingresar usuario
+        if (auxn.isEmpty()){
+
+            Toast.makeText(getApplicationContext(),"Ingrese un Usuario",Toast.LENGTH_LONG).show();
+        //sino... si la clave es vacia entonces te pide que coloques una clave
         }else if (auxp == null){
 
             Toast.makeText(getApplicationContext(),"Ingrese una contraseña",Toast.LENGTH_LONG).show();
 
+        //si la clave de confirmacion es igual a la clave entonces guardalo en el sistema
         }else if (auxp.equals(auxc)){
 
-            /*al cumplirse todas las validaciones procedemos a guardar el usuario en la tabla correspondiente*/
-            /*objeto de tipo clase donde estamos guardando*/
-            /*instanciamos y llenamos con getAplo... nombre null xq ya lo tenemos en la BASE DE DATOS
-            * null por q factory no importa y 1 el tipo de version*/
+
+
+            //al cumplirse todas las validaciones procedemos a guardar el usuario en la tabla correspondiente
+            //objeto de tipo clase donde estamos guardando
+            //instanciamos y llenamos con getAplo... nombre null xq ya lo tenemos en la BASE DE DATOS
+            // null por q factory no importa y 1 el tipo de version
             AyudaBD db = new AyudaBD(getApplicationContext(),null,null,1);
 
-            /*llenamos el metodo para enviar los parametros */
+            //llenamos el metodo para enviar los parametros
             String paquete = db.guardar(auxn,auxp);
-            /*mostramos el mensaje si no se cargobien la BD*/
+            //mostramos el mensaje si no se cargobien la BD
             Toast.makeText(getApplicationContext(),paquete,Toast.LENGTH_SHORT).show();
 
-            /* de todo ser positivo entonces limpiamos los campos */
+            // de ser positivo entonces limpiamos los campos
             aetidr.setText("");
             aetpassr.setText("");
             aetpasscr.setText("");
-            /*mostramos mensaje de aceptacion y volvemos a la pantalla principal*/
+            //mostramos mensaje de aceptacion y volvemos a la pantalla principal
 
             Toast.makeText(getApplicationContext(),"Usuario creado de forma exitosa",Toast.LENGTH_LONG).show();
 
@@ -68,15 +139,17 @@
 
         }else {
             Toast.makeText(getApplicationContext(),"La confirmacion no coincide",Toast.LENGTH_LONG).show();
-        }
+        }*/
 
-    }
 
+
     /*accion para el boton cancelar o regresar*/
 
     public void onClickRegresar (View view){
         Intent i = new Intent(this,MainActivity.class);
         startActivity(i);
     }
+
+
 
 }
Index: app/src/main/java/company/viral/organizadorjec/SQLite.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/company/viral/organizadorjec/SQLite.java	(date 1479146651000)
+++ app/src/main/java/company/viral/organizadorjec/SQLite.java	(date 1480449937000)
@@ -18,14 +18,17 @@
     //aqui se crea la tabla...
     @Override
     public void onCreate(SQLiteDatabase db) {
-        db.execSQL("create table usuarios (usuario text, password integer)");
+        db.execSQL("create table usuarios (id integer primary key autoincremet, usuario text primary key, clave text)");
+        db.execSQL("insert into usuarios values('0','admin','admin')");
 
+
     }
 
     @Override
     public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
-        db.execSQL("drop table if exists usuarios");
-        db.execSQL("create table usuarios (usuario text, password integer");
+        db.execSQL("create table usuarios (id integer primary key autoincremet, usuario text primary key, clave text)");
+        db.execSQL("insert into usuarios values('0','admin','admin')");
+
 
     }
 }
Index: app/src/main/res/layout/activity_menusito.xml
===================================================================
--- app/src/main/res/layout/activity_menusito.xml	(date 1479146651000)
+++ app/src/main/res/layout/activity_menu_central.xml	(date 1480449937000)
@@ -8,18 +8,20 @@
     android:fitsSystemWindows="true"
     tools:openDrawer="start">
 
+
     <include
-        layout="@layout/app_bar_menusito"
+        layout="@layout/app_bar_menu_central"
         android:layout_width="match_parent"
         android:layout_height="match_parent" />
 
+
     <android.support.design.widget.NavigationView
         android:id="@+id/nav_view"
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
         android:layout_gravity="start"
         android:fitsSystemWindows="true"
-        app:headerLayout="@layout/nav_header_menusito"
-        app:menu="@menu/activity_menusito_drawer" />
+        app:headerLayout="@layout/nav_header_menu_central"
+        app:menu="@menu/activity_menu_central_drawer" />
 
 </android.support.v4.widget.DrawerLayout>
Index: app/src/main/res/layout/app_bar_menusito.xml
===================================================================
--- app/src/main/res/layout/app_bar_menusito.xml	(date 1479146651000)
+++ app/src/main/res/layout/app_bar_menu_central.xml	(date 1480449937000)
@@ -5,7 +5,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fitsSystemWindows="true"
-    tools:context="company.viral.organizadorjec.Menusito">
+    tools:context="company.viral.organizadorjec.MenuCentral">
 
     <android.support.design.widget.AppBarLayout
         android:layout_width="match_parent"
@@ -21,7 +21,7 @@
 
     </android.support.design.widget.AppBarLayout>
 
-    <include layout="@layout/content_menusito" />
+    <include layout="@layout/content_menu_central" />
 
     <android.support.design.widget.FloatingActionButton
         android:id="@+id/fab"
Index: app/src/main/res/layout/content_menusito.xml
===================================================================
--- app/src/main/res/layout/content_menusito.xml	(date 1479146651000)
+++ app/src/main/res/layout/content_menu_central.xml	(date 1480449937000)
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/content_menusito"
+    android:id="@+id/contenedor"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:paddingBottom="@dimen/activity_vertical_margin"
@@ -10,7 +11,6 @@
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
     app:layout_behavior="@string/appbar_scrolling_view_behavior"
-    tools:context="company.viral.organizadorjec.Menusito"
-    tools:showIn="@layout/app_bar_menusito">
-
+    tools:context="company.viral.organizadorjec.MenuCentral"
+    tools:showIn="@layout/app_bar_menu_central">
-</RelativeLayout>
+ </RelativeLayout>
Index: app/src/main/res/layout/nav_header_menusito.xml
===================================================================
--- app/src/main/res/layout/nav_header_menusito.xml	(date 1479146651000)
+++ app/src/main/res/layout/nav_header_menu_central.xml	(date 1480449937000)
@@ -23,13 +23,13 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:paddingTop="@dimen/nav_header_vertical_spacing"
-        android:text="Organizador JEC"
+        android:text="Android Studio"
         android:textAppearance="@style/TextAppearance.AppCompat.Body1" />
 
     <TextView
         android:id="@+id/textView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Bienvenido" />
+        android:text="android.studio@android.com" />
 
 </LinearLayout>
Index: app/src/main/res/menu/activity_menu_drawer.xml
===================================================================
--- app/src/main/res/menu/activity_menu_drawer.xml	(date 1479146651000)
+++ app/src/main/res/menu/activity_menu_central_drawer.xml	(date 1480449937000)
@@ -5,31 +5,31 @@
         <item
             android:id="@+id/nav_camera"
             android:icon="@drawable/ic_menu_camera"
-            android:title="Import" />
+            android:title="Inico" />
         <item
             android:id="@+id/nav_gallery"
             android:icon="@drawable/ic_menu_gallery"
-            android:title="Gallery" />
+            android:title="Profesores" />
         <item
             android:id="@+id/nav_slideshow"
             android:icon="@drawable/ic_menu_slideshow"
-            android:title="Slideshow" />
+            android:title="Calendario" />
         <item
             android:id="@+id/nav_manage"
             android:icon="@drawable/ic_menu_manage"
-            android:title="Tools" />
+            android:title="Configuración" />
     </group>
 
-    <item android:title="Communicate">
+    <item android:title="Usuario">
         <menu>
             <item
                 android:id="@+id/nav_share"
                 android:icon="@drawable/ic_menu_share"
-                android:title="Share" />
+                android:title="Perfil" />
             <item
                 android:id="@+id/nav_send"
                 android:icon="@drawable/ic_menu_send"
-                android:title="Send" />
+                android:title="Salir" />
         </menu>
     </item>
 
Index: app/src/main/res/menu/activity_menusito_drawer.xml
===================================================================
--- app/src/main/res/menu/activity_menusito_drawer.xml	(date 1479146651000)
+++ app/src/main/res/menu/activity_menusito_drawer.xml	(date 1479146651000)
@@ -1,38 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
-
-    <group android:checkableBehavior="single">
-        <item
-            android:id="@+id/nav_camera"
-            android:icon="@drawable/icon_person"
-            android:title="Principal" />
-        <item
-            android:id="@+id/nav_gallery"
-            android:icon="@drawable/ic_menu_gallery"
-            android:title="Academia" />
-        <item
-            android:id="@+id/nav_slideshow"
-            android:icon="@drawable/ic_menu_slideshow"
-            android:title="Almanaque" />
-        <item
-            android:id="@+id/nav_manage"
-            android:icon="@drawable/ic_menu_send"
-            android:title="Perfil" />
-    </group>
-
-
-
-    <item android:title="Communicate">
-        <menu>
-            <item
-                android:id="@+id/nav_share"
-                android:icon="@drawable/ic_menu_share"
-                android:title="Configuracion" />
-            <item
-                android:id="@+id/nav_send"
-                android:icon="@drawable/ic_menu_manage"
-                android:title="Cerrar Sesion" />
-        </menu>
-    </item>
-
-</menu>
Index: app/src/main/res/menu/menu.xml
===================================================================
--- app/src/main/res/menu/menu.xml	(date 1479146651000)
+++ app/src/main/res/menu/menu_central.xml	(date 1480449937000)
@@ -1,0 +1,0 @@
Index: app/src/main/res/menu/menusito.xml
===================================================================
--- app/src/main/res/menu/menusito.xml	(date 1479146651000)
+++ app/src/main/res/menu/menusito.xml	(date 1479146651000)
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<menu xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
-    <item
-        android:id="@+id/action_settings"
-        android:orderInCategory="100"
-        android:title="@string/action_settings"
-        app:showAsAction="never" />
-</menu>
Index: app/src/main/res/values/strings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/strings.xml	(date 1479146651000)
+++ app/src/main/res/values/strings.xml	(date 1480449937000)
@@ -6,5 +6,10 @@
     <string name="navigation_drawer_close">Close navigation drawer</string>
 
     <string name="action_settings">Settings</string>
-    <string name="title_activity_menusito">Menusito</string>
+    <string name="title_activity_menusito">Bienvenido</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="title_activity_menucentral">Menucentral</string>
+    <string name="title_activity_menu_central">MenuCentral</string>
 </resources>
Index: app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/AndroidManifest.xml	(date 1479146651000)
+++ app/src/main/AndroidManifest.xml	(date 1480449937000)
@@ -23,10 +23,13 @@
             </intent-filter>
         </activity>
         <activity
-            android:name=".Menusito"
-            android:label="@string/title_activity_menusito"
-            android:theme="@style/AppTheme.NoActionBar">
-        </activity>
+            android:name=".Menucentral"
+            android:label="@string/title_activity_menucentral"
+            android:theme="@style/AppTheme.NoActionBar" />
+        <activity
+            android:name=".MenuCentral"
+            android:label="@string/title_activity_menu_central"
+            android:theme="@style/AppTheme.NoActionBar"></activity>
     </application>
 
 </manifest>
\ No newline at end of file
Index: app/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/build.gradle	(date 1479146651000)
+++ app/build.gradle	(date 1480449937000)
@@ -26,5 +26,6 @@
     })
     compile 'com.android.support:appcompat-v7:24.2.1'
     compile 'com.android.support:design:24.2.1'
+    compile 'com.android.support:support-v4:24.2.1'
     testCompile 'junit:junit:4.12'
 }
